pkg/clusterresource/controller.go

```diff
@@ -315,7 +315,7 @@ func (c *controller) syncNamespace(ctx context.Context, namespace NamespaceName,
 
 					if err != nil {
 						c.metrics.TemplateUpdateErrors.Inc()
-						logger.Infof(ctx, "Failed to update resource [%+v] in namespace [%s] with err :%v",
+						logger.Warningf(ctx, "Failed to update resource [%+v] in namespace [%s] with err :%v",
```

Comment:
`%+v`, maybe that is a problem?

Reply:
how so?

Reply:
we are dumping out the entire object, is that want you intend to do?

Reply:
we're only dumping out the Kind